src/js/background.js

```diff
@@ -18,6 +18,18 @@ const loadWordBank = function(data)
     });
 };
 
+/**
+ * Turn on the plugin on installation
```

Comment:
setPowerOff is turning on the plugin??
Rip dude, maybe just name the variable `power` and true means on while false means off?